class.jetpack-modules-list-table.php

```diff
@@ -135,7 +135,7 @@ function get_views() {
 		/** This filter is already documented in class.jetpack-modules-list-table.php */
 		$modules              = apply_filters( 'jetpack_modules_list_table_items', Jetpack_Admin::init()->get_modules() );
 		$array_of_module_tags = wp_list_pluck( $modules, 'module_tags' );
-		$module_tags          = call_user_func_array( 'array_merge', $array_of_module_tags );
+		$module_tags          = call_user_func_array( 'array_merge', array_values( $array_of_module_tags ) );
```

Comment:
We could move away from `call_user_func_array()` while we're here, if we want.
```suggestion
		$module_tags          = array_merge( ...array_values( $array_of_module_tags ) );
```

Reply:
Good call, didn't think of spread.